scripts/o3de/o3de/export_project.py

```diff
@@ -95,19 +152,33 @@ def _export_script(export_script_path: pathlib.Path, project_path: pathlib.Path,
         logging.error(f"Export script path unrecognized: '{export_script_path}'. Please provide a file path to an existing python script with '.py' extension.")
         return 1
 
+    # Compute and validate the given project path is a valid path
     computed_project_path = utils.get_project_path_from_file(export_script_path, project_path)
-
     if not computed_project_path:
         if project_path:
             logging.error(f"Project path '{project_path}' is invalid: does not contain a project.json file.")
         else:
             logging.error(f"Unable to find project folder associated with file '{export_script_path}'. Please specify using --project-path, or ensure the file is inside a project folder.")
         return 1
-    
-    o3de_context = O3DEScriptExportContext(export_script_path= export_script_path,
-                                        project_path = computed_project_path,
-                                        engine_path = manifest.get_project_engine_path(computed_project_path),
-                                        args = passthru_args)
+
+    # Collect and isolate any custom cmake build arguments that may have been passed through, use `/` as the separator in the args
+    export_process_args = []
```

Comment:
If I understand this code segment correctly, you're trying to split between cmake specific arguments and exporter arguments, something like:

`export arg 1  arg 2 .. / cmake arg 1 arg 2 ..`

in that case, you may want to first find the index of the separator, if it exists, and then slice the list you want:

```
separator_string = '/'

sep_index= passthru_args.index(separator_string)
cmake_custom_build_args = passthru_args[sep_index + 1:] #slice will select relevant cmake args after separator
export_process_args = passthru_args[:sep_index] #omit the separator
```

Reply:
thats a good idea